omnibus/config/projects/dogstatsd.rb

```diff
@@ -220,3 +220,8 @@
 
 exclude '\.git*'
 exclude 'bundler\/git'
+
+if linux_target? or windows_target?
+  strip_build windows_target? || !do_package
+  debug_path '.debug'
```

Comment:
I can't fully remember how the debug package was built, but can you double-check all these symbols are also available in said debug packages 🙇

Reply:
For the agent we have a debug package, but we don't for dogstatsd and iot
I discussed this internally and the answer seemed to be that we can strip right now and if the need arises then we can provide debug symbols

Reply:
We're discussing this within agent delivery, but yeah so far the plan would be to not care about the debug packages.

Should we want to ship those, it can easily be done in a sprint